index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="background-img">
+
+    <header>
+      <div class="header-flex">
+        <div class="logo-link"><a class="logo-link" href="https://github.com/Khawanin">Khawanin</a></div>
+        <div class="union-icon"><img src="./Images/Union.png" alt="union icon"></div>
+      </div>
```

Comment:
- [ ] Kindly make sure is wrapped in a `<nav>` tag as required for the project and for semantics reasons.